components/BasicLevelTable.tsx

```diff
@@ -0,0 +1,48 @@
+import Link from 'next/link';
+import React from 'react';
+import getProfileSlug from '../helpers/getProfileSlug';
+import { EnrichedLevel } from '../models/db/level';
+import { getFormattedDifficulty } from './difficultyDisplay';
+import EnrichedLevelLink from './enrichedLevelLink';
+
+export default function BasicLevelTable({ title, levels }: {title: string, levels: EnrichedLevel[]}) {
```

Comment:
Maybe it would be better to use the level icons (`<Select ... />`), and we can call the section for "Top Levels of the Month" with the 5 best levels (or maybe top of the week is better?)

Could possibly also reduce the latest levels limit so the home page isn't oversaturated with level options? Not sure